app/views/layouts/application.html.erb

```diff
@@ -44,7 +44,9 @@
     <% end %>
   </head>
   <% unless internal_navigation? %>
-    <% cache(release_adjusted_cache_key("top-html-and-config--#{user_signed_in?}")) do %>
+  <!-- adding the Feature Flag status to the top bar is temporary to show the logo from the correct place,
+      it will be removed by latest 24/10/2021 -->
+    <% cache(release_adjusted_cache_key("top-html-and-config--#{user_signed_in?}--#{FeatureFlag.enabled?(:creator_onboarding)}")) do %>
```

Comment:
@maestromac I needed to check with you if this okay to add for the next week. When we enable the Feature Flag on to launch creator onboarding we need the logo.html.erb to update its cache as well to show the if else.

Reply:
I will remove it next Monday once we remove the Feature Flags.

Reply:
A follow up note to confirm that Mac does not see any concerns with adding this as a cache key :)